docs/changelog/2023-03.md

```diff
@@ -4,7 +4,14 @@ hide:
 ---
 # 2023-03
 
+## Add "events add" to timesketch-cli-client
+
+The new Timesketch CLI client feature allows users to add events to their Timesketch account using the `events add` command. The command takes three arguments: `message, date, timestamp-desc` and an optional field for `attributes`. The message is a short description of the event, the date is the date and time the event occured, and the attributes are a list of key-value pairs that will be associated with the event.
+
+For example, the following command would add an event to the Timesketch `timesketch events add --message "foobar-message" --date 2023-03-04T11:31:12 --timestamp-desc "test" --attributes key=value,key2=value2`:
+
+This new feature makes it easy to add events to Timesketch from the command line, which can be helpful for automating tasks or for quickly adding events when you don't have access to the Timesketch web interface.
```

Comment:
```suggestion
## Add "events add" to timesketch-cli-client

The new Timesketch CLI client feature allows users to add events to their Timesketch sketch using the `events add` command. The command takes three arguments: `message, date, timestamp-desc` and an optional argument for `attributes`. `message` is a short description of the event, `date` is the date and time the event occured (ISO formatted), and the attributes are a comma-separated list of key-value pairs that will be associated with the event.

For example, the following command would add an event to the Timesketch 

\`\`\`
timesketch events add --message "foobar-message" --date 2023-03-04T11:31:12 --timestamp-desc "test" --attributes key=value,key2=value2
\`\`\`

This new feature makes it easy to add events to Timesketch from the command line, which can be helpful for automating tasks or for quickly adding events when you don't have access to the Timesketch web interface.
```

Reply:
I had to escape the blockquotes above to make this readable. Please unescape as you see fit, and also  update the documentation in `cli-client.md`

Also, please specify what `timestamp-desc` is supposed to be used for.